resources/views/livewire/home/statistics.blade.php

```diff
@@ -126,7 +126,11 @@ class="flex overflow-hidden relative items-center pr-4 pb-3 select-none sm:py-3
                                     <span>{{ ExchangeRate::convert($fee['amount']) }}</span>
                                     <span>{{ Settings::currency() }}</span>
                                 @else
-                                    <span>{{ $fee['amount'] }}</span>
+                                    <span>{{ ExplorerNumberFormatter::currencyWithDecimals(
+                                        value: $fee['amount']->__toString(),
+                                        currency: Settings::currency(),
+                                        decimals: 2,
```

Comment:
better to not have decimals as the input we use in vault does not use partial numbers either, refer to the card